src/types/index.d.ts

```diff
@@ -1315,3 +1315,18 @@ declare module 'wa-sqlite/src/examples/tag.js' {
    */
   export function tag(sqlite3: any, db: number): (arg0: TemplateStringsArray, ...args: any[]) => Promise<object[]>;
 }
+
+/** @ignore */
+declare module 'wa-sqlite/src/examples/IDBBatchAtomicVFS.js' {
+  import * as VFS from "wa-sqlite/src/VFS.js";
+
+  export interface VFSOptions {
```

Comment:
What is the scope of the `VFSOptions` interface name? If it's global, i.e. declaring `VFSOptions` in a different module would conflict, then maybe `IDBBatchAtomicVFSOptions` would be better.

Reply:
@rhashimoto Sure, I've renamed it.